tools/create_expfile.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+echo "Searching $1 to write out expfile to $2"
+echo "#!." > $2
+find $1 -name "*.a" |xargs nm -Xany -BCpg | awk '{ if ((($2 == "T") || ($2 == "D") || ($2 == "B")) && (substr($3,1,1) != ".")) {print $3} }'  | sort -u >> $2
```

Comment:
could we clean this up a little, some missing spaces and it's quite long, maybe something like:

```
 find $1 -name "*.a" \
  | xargs nm -Xany -BCpg \
  | awk '{
      if ((($2 == "T") || ($2 == "D") || ($2 == "B")) && (substr($3,1,1) != ".")) { print $3 }
    }'  \
  | sort -u >> $2
```

Also perhaps a comment at the top of the file that this if for creating an AIX .exp file, just to give some contents to the next poor dev who goes fishing around in tools/ and stumbles on our crazy assortment of hacks